pkg/skaffold/schema/latest/config.go

```diff
@@ -22,7 +22,7 @@ import (
 	yaml "gopkg.in/yaml.v2"
 )
 
-const Version string = "skaffold/v1alpha4"
+const Version string = "skaffold/v1alpha5"
```

Comment:
Hi @cedrickring, since you're only adding new fields, it's ok to not create a new version.